components/Select/Select.themes.ts

```diff
@@ -11,35 +12,32 @@ export namespace Theme {
     selectText: Property.Color;
     selectPlaceholder: Property.Color;
     selectDisabledText: Property.Color;
-    selectDisabledBorder: Property.Color;
-    selectReadOnlyBg: Property.Color;
+    selectInvalidBorder: Property.Color;
   };
 
-  type Factory = (primaryColor?: ColorInfo) => Colors;
+  type Factory = (primaryColor: ColorInfo) => Colors;
 
-  export const getLight: Factory = () => ({
-    selectBg: '$deepBlue3',
-    selectBorder: '$deepBlue6',
-    selectFocusBg: '$deepBlue2',
-    selectFocusBorder: '$deepBlue10',
-    selectHoverBg: '$deepBlue2',
-    selectText: '$deepBlue9',
-    selectPlaceholder: '$deepBlue6',
-    selectDisabledText: '$deepBlue5',
-    selectDisabledBorder: '$deepBlue5',
-    selectReadOnlyBg: '$deepBlue3',
+  export const getLight: Factory = (primaryColor) => ({
+    selectBg: '$deepBlue1',
+    selectBorder: '$grayBlue9',
+    selectFocusBg: tinycolor('black').setAlpha(0.15).toHslString(),
+    selectFocusBorder: primaryColor.helpers.pickScale(8),
+    selectHoverBg: '$whiteA9',
+    selectText: tinycolor('black').setAlpha(0.74).toHslString(),
+    selectPlaceholder: '$blackA10',
+    selectDisabledText: tinycolor('black').setAlpha(0.35).toHslString(),
+    selectInvalidBorder: '$red9',
```

Comment:
Simply a suggestion.

It seems we want a strong link between input styles and select styles.

Why copying values, rather than importing them in some way ? I think about either importing them in this theme file, or using them directly in the `Select.tsx` file.

Do you expect this consistency might be unwanted some day ?

We could even remap props if we want to keep both themes distinct or make sure we don't change `$variables` in `Select.tsx`.

While writing a proposal, I notice it's quite annoying and maybe useless, but I hope you get the idea.
```
import InputTheme from '...'

const remapInputTheme = (inputLightOrDark) => {
selectBg: inputLightOrDark.inputBg,
...
}

export const getLight = (primaryColor) => ({
  ...remapInputTheme(InputTheme.getLight(primaryColor),
  // specific select theme colors
})
```